config/nginx.background.conf

```diff
@@ -0,0 +1,102 @@
+# this sets the user nginx will run as,
+#and the number of worker processes
+user nobody nogroup;
+worker_processes  2;
+#user www-data;
+#worker_processes auto;
+
+# setup where nginx will log errors to
+# and where the nginx process id resides
+error_log  /var/log/nginx/error.log;
+pid        /var/run/nginx.pid;
+
+events {
+  worker_connections  1024;
+  # set to on if you have more than 1 worker_processes
+  accept_mutex on;
+}
+
+http {
+  include       /etc/nginx/mime.types;
+
+  default_type application/octet-stream;
+  access_log /tmp/nginx.access.log combined;
+
+  # use the kernel sendfile
+  # sendfile        on;  # this causes over-caching because modified timestamps lost in VM
+  # prepend http headers before sendfile()
+  tcp_nopush     on;
+
+  keepalive_timeout  600;
+  tcp_nodelay        on;
+
+  gzip  on;
+  gzip_vary on;
+  gzip_min_length 500;
+
+  gzip_disable "MSIE [1-6]\.(?!.*SV1)";
+  gzip_types text/plain text/xml text/css
+     text/comma-separated-values
+     text/javascript application/x-javascript
+     application/atom+xml image/x-icon;
+
+  # configure the virtual host
+  server {
+    # replace with your domain name
+    # server_name inferno-server;
+
+    # port to listen for requests on
+    listen 80;
+
+    # maximum accepted body size of client request
+    client_max_body_size 4G;
+    # the server will close connections after this time
+    keepalive_timeout 600;
+
+    location /validator {
+      proxy_set_header X-Forwarded-For $proxy_add_x_forwarded_for;
+      proxy_set_header Host $http_host;
+      proxy_set_header X-Forwarded-Proto $scheme;
+      proxy_set_header X-Forwarded-Port $server_port;
+      proxy_redirect off;
+      proxy_set_header Connection '';
+      proxy_http_version 1.1;
+      chunked_transfer_encoding off;
+      proxy_buffering off;
+      proxy_cache off;
+
+      proxy_pass http://fhir_validator_app;
+    }
+
+    location /validatorapi/ {
+      proxy_set_header X-Forwarded-For $proxy_add_x_forwarded_for;
+      proxy_set_header Host $http_host;
+      proxy_set_header X-Forwarded-Proto $scheme;
+      proxy_set_header X-Forwarded-Port $server_port;
+      proxy_redirect off;
+      proxy_set_header Connection '';
+      proxy_http_version 1.1;
+      chunked_transfer_encoding off;
+      proxy_buffering off;
+      proxy_cache off;
+
+      proxy_pass http://validator_service:4567/;
+    }
```

Comment:
You may want to double check with someone on this because I am not entirely sure, but based on what I have seen other test kits do, and due to the fact that I believe we want to use solely the HL7 Validator Wrapper, I think these should be commented out